### DISCUSSION TITLE
No Autogen Around Seattle

### DISCUSSION THREAD

**Discussion by simmerluke** [kubilus1/autoortho#554] [General]
This is an odd one that I've only just noticed so it may have been there a while - there is no autogen in a fairly wide radius around Seattle. The satellite imagery is fine, but there are no building or trees.

Doesn't seem to affect any other areas in the sim that I can find.

I tried removing Autoortho to see what happened with the stock scenery and that seemed to be fine, no missing buildings or trees there.

Any thoughts?

**Comment by Franzopilot** [comment]
Hi, I noticed the same. When using simheaven the autogen appears. But I'm having another issue in KSEA: the runway 16R is unusable because it looks like a roller-coaster. Is someone has been able to fix that I would be so thankful.

**Comment by Jeff2122** [comment]
I think LR is masking the areas of KPDX and KSEA when AO is enabled. I don't have a fix.
